### PR TITLE
Unified site header edition based visibility

### PR DESCRIPTION
**Pull request by wladimiiir**

### CODE REVIEW

File: app/2.0.x/index.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
-title: Documentation for Kong Gateway
-is_homepage: true
+title: Documentation for {{site.ce_product_name}}
```
Comment (lena-larionova) [inline app/2.0.x/index.md:2]:
```suggestion
title: Documentation for Kong Gateway
```
Unfortunately, browser doesn't parse this so the variable is visible in the tab label.

File: app/2.0.x/index.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+subtitle: A guide for {{site.ce_product_name}}
```
Comment (lena-larionova) [inline app/2.0.x/index.md:3]: Not sure that we really have anything to put in the subtitle field. Let's just remove it from the Kong Gateway and Kong Enterprise landing pages.

File: app/enterprise/1.5.x/index.md [hunk 3]
```diff
@@ -1,7 +1,8 @@
 ---
-title: Documentation for Kong Enterprise
+title: Documentation for {{site.ee_product_name}}
```
Comment (lena-larionova) [inline app/enterprise/1.5.x/index.md:2]:
```suggestion
title: Documentation for Kong Enterprise
```

**Comment by lena-larionova** [comment]
@hutchic @darrenjennings There's a bug with this doc site banner rework where the dropdown menu and the search bar don't seem to do anything. I've exhausted my knowledge limit here trying to fix it, and Vlad is away on vacation for three weeks (we were hoping to merge this in for the 2.1 timeframe). Any ideas?

You can see the issue on any page of the preview.

**Comment by travispamaral** [comment]
@lena-larionova the best I could do for now is to make the search appear when you click the toggle on mobile. In order to support a better design more code would need to be refactored. This at least gets it working :)
image

**Comment by lena-larionova** [comment]
> This at least gets it working :)
It's usable, which is already a bit improvement :) We'll revisit the design when Vlad comes back and has more time to spend on it.

**Pull request merged**